packages/astro/src/core/errors/errors-data.ts

```diff
@@ -530,6 +530,72 @@ See https://docs.astro.build/en/guides/server-side-rendering/ for more informati
 			)} are supported for optimization.`,
 		hint: "If you do not need optimization, using an `img` tag directly instead of the `Image` component might be what you're looking for.",
 	},
+	/**
+	 * @docs
+	 * @see
+	 * - [Assets (Experimental)](https://docs.astro.build/en/guides/assets/)
+	 * @description
+	 * The `src` property, either on `getImage`'s first parameter or on the Image component needs to be either an image that has been ESM imported or a string.
```

Comment:
```suggestion
	 * An image's `src` property is not valid.  The Image component requires the `src` attribute to be either an image that has been ESM imported or a string. This is also true for the first parameter of `getImage()`.
```
Just wondering whether something like this structure maybe this breaks up all the possible combinations here vs trying to squish it all into one sentence? (Image component/	`getImage` and ESM import/string)

Reply:
Seems fine to me! Admittedly, this is not an error you're supposed to hit often, it's extra cushion if the user is trying / not intentionally doing something weird